Cython/Utility/ObjectHandling.c

```diff
@@ -2280,50 +2192,17 @@ static CYTHON_INLINE PyObject* __Pyx_PyObject_CallOneArg(PyObject *func, PyObjec
 
 /////////////// PyObjectCallOneArg ///////////////
 //@requires: PyObjectCallMethO
-//@requires: PyObjectCall
-//@requires: PyFunctionFastCall
-//@requires: PyCFunctionFastCall
-
-#if CYTHON_COMPILING_IN_CPYTHON
-static PyObject* __Pyx__PyObject_CallOneArg(PyObject *func, PyObject *arg) {
-    PyObject *result;
-    PyObject *args = PyTuple_New(1);
-    if (unlikely(!args)) return NULL;
-    Py_INCREF(arg);
-    PyTuple_SET_ITEM(args, 0, arg);
-    result = __Pyx_PyObject_Call(func, args, NULL);
-    Py_DECREF(args);
-    return result;
-}
+//@requires: PyObjectFastCall
 
 static CYTHON_INLINE PyObject* __Pyx_PyObject_CallOneArg(PyObject *func, PyObject *arg) {
-#if CYTHON_FAST_PYCALL
-    if (PyFunction_Check(func)) {
-        return __Pyx_PyFunction_FastCall(func, &arg, 1);
-    }
-#endif
-    if (likely(PyCFunction_Check(func))) {
+    if (PyCFunction_Check(func)) {
         if (likely(PyCFunction_GET_FLAGS(func) & METH_O)) {
             // fast and simple case that we are optimising for
             return __Pyx_PyObject_CallMethO(func, arg);
```

Comment:
I'm seeing a regression in the PyPy support (shows up in travis pypy3) due to this:
`ImportError: /home/travis/build/cython/cython/TEST_TMP/2/run/c/test_fstring/test_fstring.pypy3-58-x86_64-linux-gnu.so: undefined symbol: __Pyx_PyObject_CallMethO`
This code used to be guarded for CPython-only. It's probaby ok to just map this whole function to `…CallObjectArgs()` in non-CPython.